inspire_disambiguation/core/ml/models.py

```diff
@@ -596,6 +597,11 @@ def fit(self):
         )
         self.distance_estimator.fit(self.X, self.y)
 
+    # function to score data which will return the accuracy score
+    # shouldn't we concider another statistic (f1 score?)
```

Comment:
yes, that might make more sense. Do you know what metric `score` is using here?

Reply:
It looks that it's using the score method on last step of pipeline which is RandomForestClassifier here, so it's using mean accuracy

Reply:
actually, I believe the behavior of `score` makes sense, as there's no reason to interpret one of the classes (same author/different author) as true and the other as false (which I believe you need to do in order to compute an F-score).